ci/scripts/screenshots.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# LICENSE
+#
+# screenshots.sh is part of FlyveMDMAdminDashboard
```

Comment:
Review the new @Naylin15 format for headers, now the name of the file is not present to be more flexible and readable